Bio/KDTree/KDTreemodule.c

```diff
@@ -1,21 +1,7 @@
 #include <Python.h>
-#define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION
-#include <numpy/arrayobject.h>
 #include "KDTree.h"
 
 
-/* Must define Py_TYPE for Python 2.5 or older */
```

Comment:
We currently require at least Python 2.7 onwards, so unless you have a need for this (e.g. local usage outside Biopython) we can remove this define.

Reply:
I don't need it.